akka-actor/src/main/scala/akka/io/SelectionHandler.scala

```diff
@@ -112,7 +116,10 @@ private[io] object SelectionHandler {
 
     private[this] val select = new Task {
       def tryRun(): Unit = {
+        import scala.collection.JavaConverters._
+        log.warning(selector + " Selecting... " + selector.keys().asScala.map(key => "" + key.asInstanceOf[SelectionKeyImpl].interestOps()))
```

Comment:
Maybe add the `key.attachment` to the log message so that you can see the mapping from connection actor -> interestOps.